packages/cloud-core/rollup.config.mjs

```diff
@@ -11,16 +11,16 @@ import cleanup from "rollup-plugin-cleanup";
  * @type {import('rollup').RollupOptions}
  */
 export default {
-  input: "lib/index.ts",
+  input: "lib/utils/index.ts",
   output: [
     {
-      file: "dist/index.ts",
+      file: "dist/utils/index.ts",
```

Comment:
Shall we change output to `.js`? should still build with accompanying `.d.ts` file.

Reply:
Oh my.. that went totally under my radar. Thank you